kasmvnc/run.sh

```diff
@@ -159,21 +154,29 @@ else
 fi
 
 # Coder port-forwarding from dashboard only supports HTTP
```

Comment:
```suggestion
```
> [!IMPORTANT]
>Unrelated to your PR, but while we are the refactor,

This line is redundant, as Coder now supports http and https port forwarding. 
Also, the `protocol : http` is one of the `vnc|http` options and doe snot have to do anything with https.

Reply:
Can we remove these lines then?
```yaml
ssl:
  require_ssl: false
```

Reply:
If we remove this, the kasmvncserver requires specifying SSL certs. We do not need those as the connection is already secured by Coder.